content/projects/gsoc/2019/projects.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: project
+title: "Jenkins in Google Summer of Code 2018"
```

Comment:
I am confused about this whole page, it talks about 2018 projects, but it's in the 2019 folder.